.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,16 +6,16 @@
 
 ## Pre-launch Checklist
 
-- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
 - [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
+- [ ] I signed the [CLA].
+- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
 - [ ] I listed at least one issue that this PR fixes in the description above.
-- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
 - [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
 - [ ] I updated CHANGELOG.md to add a description of the change.
```

Comment:
I would probably put these two last.

Reply:
Well until you update the changelog, the tests can't pass, so I figured the tests one had to go last, and then for thematic consistency I put the other test one with it...

Reply:
There's only a failure from not updating the CHANGELOG of you update the version, so as long as these two stay together it won't affect tests.

Reply:
Surely the checkbox for "all the tests pass" has to go last though? Since as soon as you make any change the tests have to run again, so you can't know if the tests have passed yet? Seems like by definition you have to check the "changelog" one before the "tests" one...

Anyway I don't feel strongly about this at all, I'm happy to defer to whatever order you'd like to see. Let me know what order you want it in and I'll land that. :-)

Reply:
I guess that makes sense. LGTM as-is.